jib-core/src/test/java/com/google/cloud/tools/jib/image/json/JsonToImageTranslatorTest.java

```diff
@@ -148,12 +148,12 @@ public void testVolumeMapToList() throws BadContainerConfigurationFormatExceptio
             AbsoluteUnixPath.get("/var/log/my-app-logs"));
     Assert.assertEquals(expected, JsonToImageTranslator.volumeMapToSet(input));
 
-    ImmutableList<Map<String, Map<?, ?>>> badInputs =
+    ImmutableList<Map<String, Map<String, String>>> badInputs =
         ImmutableList.of(
             ImmutableMap.of("var/job-result-data", ImmutableMap.of()),
             ImmutableMap.of("log", ImmutableMap.of()),
             ImmutableMap.of("C:/udp", ImmutableMap.of()));
-    for (Map<String, Map<?, ?>> badInput : badInputs) {
+    for (Map<String, Map<String, String>> badInput : badInputs) {
```

Comment:
Maybe we can keep the original code for the cases like in these tests. The sonarcloud is only talking about a wildcard in a return type of a method. I think if the type is obvious in a small local scope, I think it reduces the boilerplate and cognitive load (as long as sonarcloud doesn't complain).

Reply:
That's a good observation. Unfortunately, `volumeMapToSet` takes in `containerConfigurationTemplate.getContainerVolumes` as an input in line 201 of `JsonToImageTranslator` so it requires the parameter type to be Map<String, Map<String, String>> . The code doesn't compile when we give it an input that's of type `Map<String, Map<?,?>>`